scripts/eqtl_hail_batch/generate_eqtl_spearman.py

```diff
@@ -28,6 +29,7 @@
 TOB_WGS = dataset_path('mt/v7.mt/')
 FREQ_TABLE = dataset_path('joint-calling/v7/variant_qc/frequencies.ht/', 'analysis')
 VEP_ANNOTATION = dataset_path('tob_wgs_vep/v1/vep105_GRCh38.mt/')
+GENCODE_GTF = 'gs://cpg-reference/gencode/gencode.v38.annotation.gtf.gz'
```

Comment:
Unrelated to this PR, but noting that we should likely rerun this with VEP 104?

Reply:
I'm happy to rerun VEP using v104 if that's what's decided upon